js/app.js

```diff
@@ -27,6 +81,26 @@ function createDefaultSnippetPreview() {
 	} );
 }
 
+function checkArguments( args ) {
```

Comment:
Please document this function. Also use the `@private` flag to indicate that this is a private function.